### PR TITLE
Update security-customization.adoc.po

### PR DESCRIPTION
**Pull request by freakse-sa**

### CODE REVIEW

File: l10n/po/ja_JP/_guides/security-customization.adoc.po [hunk 1]
```diff
@@ -166,39 +165,34 @@ msgstr "}\n"
...
 
 #. type: Plain text
 #: upstream/_guides/security-customization.adoc:76
-#, fuzzy
 msgid "More than one `HttpAuthenticationMechanism` can be combined, for example, the built-in `Basic` or `JWT` mechanism provided by `quarkus-smallrye-jwt` has to be used to verify the service clients credentials passed as the HTTP `Authorization` `Basic` or `Bearer` scheme values while the `Authorization Code` mechanism provided by `quarkus-oidc` has to be used to authenticate the users with Keycloak or other OpenID Connect providers."
-msgstr "例えば、HTTP `Authorization` `Basic` または `Bearer` スキーム値として渡されたサービスクライアントの資格情報を検証するために、 `quarkus-smallrye-jwt` によって提供される組み込みの `Basic` または `JWT` 機構を使用しなければならない一方で、 `quarkus-oidc` によって提供される `Authorization Code` 機構は `Keycloak` または他の `OpenId Connect` プロバイダとユーザを認証するために使用しなければならない、 複数の `HttpAuthenticationMechanism` が組み合わせられることができる。"
+msgstr "複数の `HttpAuthenticationMechanism` は組み合わせて使っていくことができます。例えば、HTTP `Authorization` `Basic` または `Bearer` スキーム値として渡されたサービスクライアントの資格情報を検証するためには、 `quarkus-smallrye-jwt` によって提供される `Basic` または `JWT` 機構を使う必要がありますし、Keycloak やその他の OpenID Connect プロバイダでユーザを認証するには `quarkus-oidc` の `Authorization Code` メカニズムを使用していく必要があります。"
```
Comment (ynojima) [inline l10n/po/ja_JP/_guides/security-customization.adoc.po:175]:
```suggestion
msgstr "複数の `HttpAuthenticationMechanism` は組み合わせて使っていくことができます。例えば、HTTP `Authorization` `Basic` または `Bearer` スキーム値として渡されたサービスクライアントの資格情報を検証するためには、 `quarkus-smallrye-jwt` によって提供される `Basic` または `JWT` メカニズムを使う必要がありますし、Keycloak やその他の OpenID Connect プロバイダでユーザを認証するには `quarkus-oidc` の `Authorization Code` メカニズムを使用していく必要があります。"
```

File: l10n/po/ja_JP/_guides/security-customization.adoc.po [hunk 2]
```diff
@@ -1066,9 +1041,8 @@ msgstr ""
 
 #. type: Plain text
 #: upstream/_guides/security-customization.adoc:587
-#, fuzzy
 msgid "This will allow you to propagate the identity throughout the reactive callbacks. You also need to make sure you are using an executor that is capable of propagating the identity (e.g. no `CompletableFuture.supplyAsync`), to make sure that Quarkus can propagate it. For more information see the xref:context-propagation.adoc[Context Propagation Guide]."
-msgstr "これにより、リアクティブコールバック全体にアイデンティティを伝搬させることができます。また、Quarkusがアイデンティティを伝搬できるようにするために、アイデンティティを伝搬できるエグゼキュータを使用していることを確認する必要があります（例えば、 `CompletableFuture.supplyAsync` は使用していません）。詳細については、 link:context-propagation.html[Context Propagation Guide]を参照してください。"
+msgstr "これにより、リアクティブコールバック全体にアイデンティティを伝搬させることができます。また、Quarkusがアイデンティティを伝搬できることを確かにするために、アイデンティティを伝搬できるエグゼキュータを使用していることを確認する必要があります（例:  `CompletableFuture.supplyAsync`  を使用しない場合）。詳細については、 link:context-propagation.html[Context Propagation Guide]を参照してください。"
```
Comment (ynojima) [inline l10n/po/ja_JP/_guides/security-customization.adoc.po:1045]:
```suggestion
msgstr "これにより、リアクティブコールバック全体にアイデンティティを伝搬させることができます。また、Quarkusがアイデンティティを伝搬できることを確かにするために、アイデンティティを伝搬できるエグゼキューターを使用していることを確認する必要があります（例:  `CompletableFuture.supplyAsync`  を使用しない場合）。詳細については、 link:context-propagation.html[Context Propagation Guide]を参照してください。"
```

**Comment by ynojima** [comment]
Thank you for your pull-request, and Sorry for belated review, I merged it into `main`.

**Comment by freakse-sa** [comment]
Thank you for your review and merge.

**Pull request merged**